tests/test_transaction.py

```diff
@@ -76,6 +76,22 @@ def test_create_transaction_with_splits_percent_too_high():
         )
 
 
+def test_split_overshoot_issue63():
```

Comment:
As you might see with other tests, the issue is typically referenced in the docstring, not the function name